config/templates/cluster.yaml

```diff
@@ -469,6 +469,12 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 #     # See https://github.com/coreos/kube-aws/issues/230 for more information.
 #     rbac:
 #       enabled: true
+#    # When enabled it will grant sts:assumeRole permission to everyrole in the IAM Worker Role.
+#    # if you specify workerNamedIAMRole the role created will not suffix the random id at the end
+#    # which is useful to update trust policy in target IAM roles that container wants to use through kube2iam.
+#    kube2IamSupport:
+#      enabled: true
+#      workerNamedIAMRole: "yournamedrole"
```

Comment:
@pieterlange if you have a target role for kube2iam ,let say role A, if role A has a trust policy for role R set in cluster.yaml the cluster recreation does not need to retriever a trust policy update